app/domain/domain.go

```diff
@@ -0,0 +1,25 @@
+package domain
+
+import (
+	"os"
+)
+
+func GetEnv(environment string, default_value string) string {
```

Comment:
в Го используют CamelCase
```suggestion
func GetEnv(environment string, defaultValue string) string {
```